recipe/meta.yaml

```diff
@@ -18,38 +21,54 @@ outputs:
     build:
       noarch: python
     requirements:
-      build:
+      host:
         - python >=3.6
         - pip
       run:
         - python >=3.6
-        - numpy >=1.9.1
-        - scipy >=0.14
-        - pyyaml
-        - h5py
-        - tensorflow >=2.2
+        # - numpy >=1.9.1
+        # - scipy >=0.14
+        # - pyyaml
+        # - h5py
+        # - tensorflow >=2.2
       # add run_constrained to prevent installation of mismatched keras and
       # keras-base packages
       run_constrained:
-        - keras {{ version }}
```

Comment:
I consider the run constrain of tensorflow as wrong.  Instead we need to make sure that base and keras do not run out of sync

Reply:
The constrains for tensorflow were removed, only the constraint for keras was left on the recipe
https://github.com/AnacondaRecipes/keras-feedstock/blob/update_to_2.6.0/recipe/meta.yaml#L39